### PR TITLE
script related to MBA method

### PR DESCRIPTION
**Pull request by arichelle**

### CODE REVIEW

**Comment by rmtfleming** [comment]
Great work Anne! Increasing the breadth of COBRA techniques will be well appreciated by our community.

File: src/dataIntegration/transcriptomics/MBA/MBA.m [hunk 1]
```diff
@@ -0,0 +1,73 @@
+function tissueModel = MBA(model, expressionRxns, threshold_medium, threshold_high, tol, core)
+% Uses the MBA algorithm (`Jerby et al., 2010`) to extract a context
```
Comment (tpfau) [inline src/dataIntegration/transcriptomics/MBA/MBA.m]: As with fastcore: I would suggest to handle the setting of the different groups outside of the algorithm (higher flexibility, users can choose what they want). 
i.e. change the header to 
```'
function tissueModel = MBA(model,medium_set, high_set, tol)
```
This is closer to the description of the algorithm in the paper (my opinion).

**Comment by laurentheirendt** [comment]
@arichelle, @jacekwachowiak already formatted the documented for you in this PR. May you change the documentation formatting back to what @jacekwachowiak did? Thank you.

**Comment by tpfau** [comment]
Looks fine

**Comment by laurentheirendt** [comment]
Thanks @arichelle 🎉

**Pull request merged**